frontend/app/src/components/Sidebar/styled-components.ts

```diff
@@ -264,57 +258,108 @@ export const StyledSidebarContent = styled.div<StyledSidebarContentProps>(
   })
 )
 
-export const StyledSidebarCloseButton = styled.div(({ theme }) => ({
+export const StyledResizeHandle = styled.div(({ theme }) => ({
   position: "absolute",
-  top: theme.spacing.xs,
-  right: theme.spacing.twoXS,
-  zIndex: 1,
+  width: "8px",
+  height: "100%",
+  cursor: "col-resize",
+  zIndex: theme.zIndices.sidebarMobile,
 
-  "&:hover button": {
-    backgroundColor: transparentize(theme.colors.fadedText60, 0.5),
+  "&:hover": {
+    backgroundImage: `linear-gradient(to right, transparent 20%, ${theme.colors.fadedText20} 28%, transparent 36%)`,
   },
+}))
 
-  [`@media print`]: {
-    display: "none",
+export const StyledSidebarHeaderContainer = styled.div(({ theme }) => ({
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "start",
+  padding: `${theme.spacing.xl} ${theme.spacing.twoXL} ${theme.spacing.twoXL} ${theme.spacing.twoXL}`,
+}))
+
+export const StyledLogoLink = styled.a(({}) => ({
+  maxWidth: "15rem",
+  "&:hover": {
+    opacity: "0.7",
   },
 }))
 
-export interface StyledSidebarCollapsedControlProps {
+export const StyledLogo = styled.img(({ theme }) => ({
+  height: "1.5rem",
+  maxWidth: "15rem",
+  margin: "0.25rem 0 0.25rem 0",
+  zIndex: theme.zIndices.header,
+}))
+
+export const StyledNoLogoSpacer = styled.div(({}) => ({
+  height: "2.0rem",
+  marginRight: "-1rem",
```

Comment:
Can we have a comment for the negative margin? It's a bit weird, so would love just an explanation here. Must be compensating for something?

Reply:
Played around and managed to remove 👍🏼